hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -1298,64 +1298,21 @@ private void deleteInternal(Container container, boolean force)
         // If the container is not empty, it should not be deleted unless the
         // container is being forcefully deleted (which happens when
         // container is unhealthy or over-replicated).
-        if (container.getContainerData().getBlockCount() != 0) {
-          metrics.incContainerDeleteFailedBlockCountNotZero();
+        if (!container.isEmpty()) {
```

Comment:
Should we change the name of this method so it is not confused with `KeyValueContainerData#isEmpty`? Maybe changing this method to `hasBlocks` or something like that?

Reply:
Updated to `hasBlocks`